components/style/selector_matching.rs

```diff
@@ -146,7 +146,7 @@ impl<Impl: SelectorImplExt> Stylist<Impl> {
         }
 
         if self.quirks_mode {
-            self.add_stylesheet(&Impl::get_quirks_mode_stylesheet());
+            Impl::get_quirks_mode_stylesheet().map(|s| self.add_stylesheet(s));
```

Comment:
Not too relevant, but I prefer the `if let Some(..)` idiom for things that have side effects and which return result you don't need.